docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '2'
+
+services:
+
+
+    jekyll:
+        build:
```

Comment:
It would be nice to provide two versions of this guy, one that is built locally and one that is pulling the image out of the hub.  For really vanilla jekyll projects we can build a single image that a lot can use.  We can also open the possibility of creating a code for dc themed jekyll image.

We can do this separately though.

Reply:
There was a Jekyll image in the previous docs, i.e. current version. I think we might have used that to start, then realized we had conflicts, then debugged a couple small changes to end up with this. 

On my machine at least, it only takes a 20-30 seconds to build the image. I was thinking that it would be worth simplifying by only providing an autobuilt version that will work for both vanilla and complex projects so it's not something project leads need to learn about choosing between or upgrading from later. 

What do you see as the downsides of only providing an autobuilt version?

Reply:
The only reason for the prebuilt version is mostly just consistency, especially if they get deployed to production.  But since we're not doing that now I think this is fine.